lessons/11-basic-data-structs/assignments/account-generator.md

```diff
@@ -0,0 +1,28 @@
+# Student Account Generator
+
+We are going to write a program that generates some student information. This exercise will practice our skills with strings, random numbers, arrays and iterators.
+
+## Requirements
+
+1. You will start by creating three arrays:
+    - An array that will contain student names
+    - An array that will store student ID numbers
+    - An array that will contain student email addresses    
```

Comment:
I would add here: Make sure to name your variables appropriate so that it is easy to deduce the information stored in each array and that this information is not singular. e.g. `flowers = ["lily", "rose", "gardenia", "daisy"] vs. flower = "tulip"`